.circleci/config.yml

```diff
@@ -1,8 +1,10 @@
+# TODO setup default tap-tester slack orb
+# TODO create a card to add pylint
```

Comment:
We should create the ticket for devs to implement pylint on tap code and ref it in this file.
Also we should add the default slack orb - we can do together.